reactor-core/src/test/java/reactor/core/publisher/FluxTimedTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2011-Present VMware Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.core.publisher;
+
+import java.time.Duration;
+import java.util.function.Consumer;
+
+import org.junit.jupiter.api.Test;
+
+import reactor.test.StepVerifier;
+import reactor.util.function.Tuple2;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class FluxTimedTest {
+
+	@Test
+	void compareTimedElapsedNanosToElapsedT1() {
```

Comment:
Could this use virtual time to avoid losing ~1s each?

Reply:
reworked the test to use virtual time (through explicit use of a `VirtualTimeScheduler`)